report_thesis/src/sections/background.tex

```diff
@@ -226,28 +226,40 @@ \subsubsection{Extra Trees Regressor (ETR)}
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
 \subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
-In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
-\gls{gbr} is an ensemble learning method that sequentially construct multiple decision trees, each one correcting the errors of the previous one, using gradient descent and boosting techniques.
-Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions.
-This ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+Continuing with the discussion of ensemble learning models, we introduce \gls{gbr} based on Chapter 10.10 of \citet{hastie_elements}.
+
+Gradient Boosting is a powerful machine learning technique used for regression and classification tasks.
+The core idea is to build a model from weak learners, typically decision trees, in a sequential manner.
+Each subsequent model aims to correct the errors of its predecessor, resulting in a strong predictive model.
+
+\gls{gbr} is a specific implementation of gradient boosting for regression tasks.
+\gls{gbr} builds a series of decision trees sequentially, where each tree tries to reduce the errors made by the previous one.
```

Comment:
```suggestion
Gradient Boosting is a powerful machine learning technique used for regression and classification tasks, that builds strong predictive models from a sequence of weaker ones, typically decision trees.
The core idea of this method lies in its sequential approach, where each tree is constructed to address and minimize the residuals or prediction errors of its predecessor.
Within this framework, \gls{gbr} applies this principle specifically to regression tasks, strategically refining each subsequent model to enhance overall accuracy.
```

My goal with this suggestion is to avoid a bit of the repetition related to "correcting the errors of the previous one". 
What do you think about this idea? I thought, since gbr builds on this concept, this might provide a better flow.